e2e/staticpvc.go

```diff
@@ -189,6 +189,109 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock, checkI
 	return err
 }
 
+func validateRBDStaticMigrationPV(f *framework.Framework, appPath string, isBlock, checkImgFeat bool) error {
+	opt := make(map[string]string)
+	var (
+		rbdImageName = "test-static-pv"
+		pvName       = "pv-name"
+		pvcName      = "pvc-name"
+		namespace    = f.UniqueName
+		// minikube creates default class in cluster, we need to set dummy
+		// storageclass on PV and PVC to avoid storageclass name mismatch
+		sc = "storage-class"
+	)
+
+	c := f.ClientSet
+	mon := "rook-ceph-mon-a.rook-ceph.svc.cluster.local:6789"
```

Comment:
please retrieve mon from the ceph cluster instead of hardcoding it.

Reply:
I actually did that yeterday but it didnt go through as mon was referred here as this in the conf, regardless this is not a concern as its admin's responsibility to make sure the mapping exist. also this proves that things are working pretty good.

Reply:
regardless, also adding a TODO comment for later.

Reply:
https://github.com/ceph/ceph-csi/blob/22bb31df196ba72cb967a9f9d98e6e277315aa6f/e2e/utils.go#L69 

Mons can be fetched using this helper ?
(even though i think the service will lead to the same ip format each time.)

Reply:
yeah, will try to figure it out later as its not a priority one atleast for now.

Reply:
but why merge/push a code that will break if someone runs tests locally?

Reply:
@Madhu-1 addressed it.. thanks.